src/structures/Player.ts

```diff
@@ -10,6 +10,9 @@ function check(options: PlayerOptions) {
     throw new TypeError(
       'Player option "guild" must be present and be a non-empty string.'
     );
+  
+  if (options.region && !/^\d+$/.test(options.region))
+    throw new TypeError('Player option "region" must be a non-empty string.');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/MenuDocs/erela.js/pullRequest?prid=7585535)
```suggestion
    throw new TypeError("Player option \"region\" must be a non-empty string.");
```